install.ps1

```diff
@@ -1,84 +1,86 @@
 # Enable TLS 1.2 since it is required for connections to GitHub
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$ErrorActionPreference = "Stop"
 
-Write-Host "Beginning installation of spicetify-bloom"
-Write-Host "https://github.com/nimsandu/spicetify-bloom"
+Write-Host -Object "Beginning installation of spicetify-bloom"

```

Comment:
I personally wonder what's the use of this parameter. Does it improve anything? If yes, how? I tried to check PowerShell documentation but it's not too descriptive about it. :/

Reply:
https://poshcode.gitbook.io/powershell-practice-and-style/style-guide/naming-conventions#use-full-parameter-names.

Reply:
Fair enough, let's hope it doesn't break anything in the future.